spec/lib/presenters/show_presenter_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+require "presenters/show_presenter"
+
+RSpec.describe ShowPresenter do
+  describe "#page_title" do
+    it "is the resource's key attribute" do
+      customer = build(:customer, name: "Worf")
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
+
+      expect(presenter.page_title).to eq("Worf")
+    end
+  end
+
+  describe "#attributes" do
+    it "returns attributes defined by the dashboard" do
+      customer = build(:customer, name: "Worf")
+      allow(customer).to receive(:lifetime_value).and_return(10)
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
+
+      expect(presenter.attributes).to eq({
+        email: customer.email,
+        lifetime_value: customer.lifetime_value,
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.